changelog

```diff
@@ -1,3 +1,213 @@
+Version 0.10.1
+  * Big changes:
+    * Created an extensive storyline that takes place after the main campaign is compeleted, focusing on the open secret of the existance of the Hai. (@MasterOfGrey, @NomadicVolcano, @UnorderedSigh)
+      * Development help and fixes from many community members. (@Amazinite, @bene-dictator, @EjoThims, @warp-core)
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@bene-dictator, @Deltaspace0, @quyykk, @samrocketman, @shitwolfymakes, @Terin, @tibetiroka, @TomGoodIdea, @UnorderedSigh, @warp-core)
+      * Fixed the offer requirements for various Coalition missions that had been unable to offer under the correct circumstances. (@Arachi-Lover)
+      * Added a proper plural form to the Solar Batteries outfit. (@Terin)
+      * Fixed the displaying of the Basrem Reverse Thrusters description. (@Fzzr)
+      * Cleaned up the brown haze graphics. (@RisingLeaf)
+      * The "Puny" outfit no longer disappears after reloading the game. (@warp-core)
+      * The "Stranded field trip to <planet>" job description now mentions that there is cargo involved. (@3agleEmpire)
+      * Fixed the NPC spawn distance on a Marauder job that was using an older value, spawning ships up to three systems away instead of up to one. (@Terin)
+      * Adjusted the source filter and offer conditions on the "Local Politics" mission to offer in more appropriate locations and at more appropriate times. (@Zitchas)
+      * Removed the required reputation for landing on Zenith after completing the alternative route in the Sestor Invasion arc. (@warp-core)
+      * Corrected various jobs that were intended to require that the player had a certain outfit installed, but also accepted that outfit being in your cargo hold. (@warp-core)
+      * The "Immigrant Workers" mission can now longer have a pirate planet destination. (@Anarchist2)
+      * The "Southern Blockade" description can no longer overflow the description text box. (@Anarchist2)
+      * "Unwanted Cargo" now only triggers from inhabited worlds. (@mOctave)
+      * Various Quarg missions no longer offer on the Gegno Quarg locations, since the Quarg there can't communicate in the same way as other location. (@Saugia)
```

Comment:
```suggestion
      * Various Quarg missions no longer offer on the Gegno Quarg locations, since the Quarg there can't communicate in the same way as other locations. (@Saugia)
```